roles/setup_radvd/README.md

```diff
@@ -0,0 +1,21 @@
+# Set Up the Router ADVertisement Daemon (radvd) role
+
+The purpose of this role is to set up an infrastructure host with a basic IPv6 radvd to facilitate
+self-management of IPv6 DHCP6 IP address management in a lab subnet.
+
+This requires switch configuration to ensure they are not competing with your infrastructure host to answer requests.
+
+## Variables
+
+All the variables defined are employed exclusively in the radvd.conf Jinja template to create the configuration file.
+
+Two required variables have no default, and must be defined in inventory to make use of this role:
+
+- `baremetal_bridge` This defines the interface radvd will listen on and is used extensively elswhere in this collection. 
```

Comment:
I may need help / consensus on those two variables. They are both meant to be shared from the rest of the inventory; they serve as global environment-wide variables.

I picked `baremetal_bridge` because it is used by the node_prep role, and this role serves a parallel purpose, to prepare the infrastructure host node for deployments.

I have my own role for dnsmasq setup which I'm not ready to bring in yet (it is too hard-coded to my environment still) and I am using `dnsmasq_ipv6_network` in that role to represent the network address for the environment.

I could replace both variables with `setup_radvd_` versions, so calls to the role would have to provide translations from, e.g. `baremetal_bridge` to `setup_radvd_baremetal_bridge` but that seems clunky to me.

Reply:
Interesting suggestion, but I think I prefer clunky to pioneering a new naming scheme in the collection.

I renamed `baremetal_bridge` and `dnsmasq_ipv6_network` to match other role vars with `setup_radvd` as the prefix.

Reply:
/hold
I just realized my ipv6_network variable would serve better as a cidr -- my version hard-codes things it should not.

Reply:
Sounds good, I wrote the lint rules with possible global variables in mind fwiw https://github.com/redhatci/ansible-collection-redhatci-ocp/blob/main/hack/rules/redhat_ci.py#L96-L103

Reply:
I don't guess the slash hold command worked here. I just pushed the changes I was talking about, and briefly tested in my own setup.

Reply:
I can do that, I have been back-porting the role to my own environment in preparation.